--- contrib/ports/Common.allports.mk~	2020-05-06 14:08:35.240359551 +0200
+++ contrib/ports/Common.allports.mk	2020-05-06 14:11:22.996268891 +0200
@@ -33,7 +33,7 @@
 #CC=clang
 CCDEP?=$(CC)
 
-CFLAGS+=-g -DLWIP_DEBUG -Wall -pedantic -Werror \
+CFLAGS+=-g -DLWIP_DEBUG -Wall \
 	-Wparentheses -Wsequence-point -Wswitch-default \
 	-Wextra -Wundef -Wshadow -Wpointer-arith -Wcast-qual \
 	-Wc++-compat -Wwrite-strings -Wold-style-definition -Wcast-align \
